e2e-polybft/e2e/bridge_test.go

```diff
@@ -1374,6 +1374,12 @@ func TestE2E_Bridge_NonMintableERC20Token_WithPremine(t *testing.T) {
 			false)
 		require.NoError(t, err)
 
+		currentBlock, err := validatorSrv.JSONRPC().GetBlockByNumber(jsonrpc.LatestBlockNumber, false)
+		require.NoError(t, err)
+
+		// Wait for 1 block before getting expected child balance
+		require.NoError(t, cluster.WaitForBlock(currentBlock.Header.Number+1, time.Second*5))
```

Comment:
Optional nitpick:
what do you think about extracting this part into some inline function, that can be invoked on both occasions? Maybe we could spare 2-3 lines of code, unsure do you think it is worth it?